conf/application.conf

```diff
@@ -38,7 +38,7 @@ play.modules.enabled += "UpscanStubModule"
 
 play.http.errorHandler = "uk.gov.hmrc.play.bootstrap.http.JsonErrorHandler"
 play.http.filters = "filters.ExpandedMicroServiceFilters"
-
+play.http.parser.maxDiskBuffer = 110MB app conf
```

Comment:
I don't think the app cons should be here

Reply:
app conf has been removed